File: 백준/Silver/7596. MP3 Songs/MP3 Songs.py
cnt = 1
while 1:
    N = int(input())
    if not N:
        break

    arr = []
    for _ in range(N):
        arr.append(input())

    arr.sort()
    print(cnt)
    for lst in arr:
        print(lst)

    cnt += 1

File: 백준/Silver/7596. MP3 Songs/MP3 Songs.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;
public class Main {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int cnt = 1;
		while(true) {
			int N = Integer.parseInt(br.readLine());
			if(N == 0) {
				break;
			}
			String [] arr = new String[N];
			for (int i = 0; i < N; i++) {
				String lst = br.readLine();
				arr[i] = lst;
			}
			Arrays.sort(arr);
			System.out.println(cnt++);
			for (String list: arr ) {
				System.out.println(list);
			}
		}
	}
}